addons/packages/grafana/7.5.11/bundle/config/certificates.lib.yaml

```
@@ -0,0 +1,178 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:template", "template")
+#@ load("@ytt:yaml", "yaml")
+
+#@ def get_issuer():
+apiVersion: cert-manager.io/v1alpha2
```

```suggestion
apiVersion: cert-manager.io/v1
```

The latest cert-manager versions have dropped `v1alpha2`

---

changed to version v1